doi/src/main/java/org/fao/geonet/doi/client/DoiManager.java

```diff
@@ -115,6 +119,11 @@ public void loadConfig() {
             String password = sm.getValue(DoiSettings.SETTING_PUBLICATION_DOI_DOIPASSWORD);
 
             doiPrefix = sm.getValue(DoiSettings.SETTING_PUBLICATION_DOI_DOIKEY);
+            doiPattern = sm.getValue(DoiSettings.SETTING_PUBLICATION_DOI_DOIPATTERN);
+            if (StringUtils.isEmpty(doiPattern)) {
```

Comment:
Maybe to use `StringUtils. defaultString`

Reply:
Improved. I let you double check. Thanks